### DISCUSSION TITLE
Exe Installer, copy from zip, or in-app update? Updating from v.2.91 to v.2.99

### DISCUSSION THREAD

**Discussion by eightball-unlock** [LesFerch/WinSetView#131] [Q&A] [answered]
Hi, still loving the program but I've been behind on updates.

As in the title, should I use the EXE package installer, overwrite existing files using the new zip file, or click the Update button in the program?

Thanks, Z

**Comment by LesFerch** [comment, marked as answer]
The Update button in the program just takes you to the web page. From there it's up to you to choose the Zip or the Exe. Either method will work fine. If you use the Exe  package installer, just be sure to point it to your existing WinSetView folder. It will not overwrite your settings. Or unzip the Zip file and copy the contents to your existing WinSetView folder, allowing it to overwrite. Again, your settings are safe because, unlike earlier versions, there is no AppData folder in the archive.

**Reply by eightball-unlock** [reply]
Thanks again!
